funnel/utils/markdown/mdit_plugins/anchors_ext.py

```diff
@@ -0,0 +1,45 @@
+"""MDIT plugin to modify the token stream output of mdit-py-plugin heading-anchors."""
+
+from __future__ import annotations
+
+from markdown_it import MarkdownIt
+from markdown_it.rules_inline import StateInline
+
+__all__ = ['anchors_extend_plugin']
+
+
+def anchors_ext(state: StateInline) -> None:
+    prev_token = None
+
+    for token in state.tokens:
+        if prev_token is None:
+            prev_token = token
+            continue
+        if token.type == 'inline' and prev_token.type == 'heading_open':  # type: ignore
```

Comment:
`type: ignore[specific_type]` needed

Reply:
Ok

Reply:
Though, the failure was statement not reachable.